astropy/io/fits/hdu/compressed.py

```diff
@@ -1362,21 +1362,6 @@ def _update_header_data(self, image_header,
             for _ in range(required_blanks - table_blanks):
                 self._header.append()
 
-    @deprecated('0.3', alternative='(refactor your code)', pending=True)
```

Comment:
I think what this really was, is that this method never should have been public in the first place and there's no reason anyone should have been using it directly.  But the message probably could have been clearer about that :)

Fortunately I don't think anyone actually *was* using this method.